resources/html/test/testPlayground.html

```diff
@@ -180,7 +180,10 @@
             Open in new window/tab
         -->
         <div id='openLinkInNewWindowDiv'>
-            <a href="onOffScreen.html" target="_blank">Open in new window</a>
```

Comment:
To create the test, I needed an element that would open a new window after a delay.